### PR TITLE
Shopify overhaul

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: assets/js/broken_redirect_list.js [hunk 1]
```diff
@@ -943,10 +943,106 @@ validurls['/docs/user_guide/engagement_tools/canvas/canvas_components/experiment
...
 validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_data_in_braze/'] = '/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/Using_Shopify/shopify_data_in_braze/'
 validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/use_cases/'] = '/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/Using_Shopify/use_cases/'
 validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/scripttag_web_sdk_integration/'] = '/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/getting_started_shopify/'
-validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_backfill/'] = '/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/overview/shopify_backfill/'
\ No newline at end of file
+validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_backfill/'] = '/docs/partners/message_orchestration/channel_extensions/ecommerce/shopify/overview/shopify_backfill/'
+=======
```
Comment (bre-fitzgerald) [inline assets/js/broken_redirect_list.js:954]: @rachel-feinberg it's this line

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/Using_Shopify/shopify_data_in_braze.md [hunk 2]
```diff
@@ -316,9 +342,479 @@ Using nested object support for custom events, Braze Shopify customers can use L
...
+| Attribute Name | Description |
+| --- | --- |
+| `shopify_tags`  | This attribute corresponds to the [customer tags](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/using-tags#tag-types) set by Shopify admins. |
+| `shopify_total_spent` | This attribute tracks the total amount spent on a store and is only supported for users imported through the [Historical Backfill]({{site.baseurl}}/partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_backfill/) feature. |
+| `shopify_order_count` | This attribute tracks the total number of orders made in a store and is only supported for users imported through the Historical Backfill feature. |
+{: .reset-td-br-1 .reset-td-br-2}
+
+{% endtab %}
+{% tab Example Payload %}
+{% subtabs global %}
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/Using_Shopify/shopify_data_in_braze.md:786]: I don't think you want this to be a global subtab, it isn't rendering until clicked right now. 

```suggestion
{% subtabs %}
```

**Review by bre-fitzgerald — APPROVED** [review]
Lgtm! Great job taking this across the line. I reviewed all the links and tabs, only found one minor issue.

**Review by lydia-xie — APPROVED** [review]
Looks fantastic! 🎉

**Pull request merged**